src/index.js

```diff
@@ -1,92 +1,24 @@
 import './style.css';
+import StorageL from './modules/storage.js';
```

Comment:
- [ ] Kindly write a function that will reorder a selected item (as drag-and-drop). This is part of the main requirement.

Reply:
Hi i think that is optional drag and drop it not neccesary , please check